include/boost/json/pointer.hpp

```diff
@@ -0,0 +1,30 @@
+//
+// Copyright (c) 2022 Dmitry Arhipov (grisumbras@yandex.ru)
+//
+// Distributed under the Boost Software License, Version 1.0. (See accompanying
+// file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+//
+// Official repository: https://github.com/boostorg/json
+//
+
+#ifndef BOOST_JSON_POINTER_HPP
+#define BOOST_JSON_POINTER_HPP
+
+#include <boost/json/string.hpp>
+#include <boost/json/system_error.hpp>
+
+BOOST_JSON_NS_BEGIN
+
+std::pair<std::size_t, string_view>
+parse_pointer_number_segment(
```

Comment:
This is the name of a public function?

Reply:
Yes. It's a helper that users need to implement their own behaviour while traversing pointers.